Tests/SwiftPrometheusTests/PrometheusMetricsTests.swift

```diff
@@ -21,6 +21,13 @@ final class PrometheusMetricsTests: XCTestCase {
         self.prom = nil
         try! self.group.syncShutdownGracefully()
     }
+
+    func testGetPrometheus() {
+        MetricsSystem.bootstrapInternal(NOOPMetricsHandler.instance)
+        XCTAssertThrowsError(try MetricsSystem.prometheus())
+        MetricsSystem.bootstrapInternal(PrometheusMetricsFactory(client: self.prom))
```

Comment:
Not sure I love that style but I guess this was introduced some time recently for some reason?

Reply:
Yeah, see #46 and #44 for full context.

Using `PrometheusClient` directly limited some options :)

Reply:
I see, so it's for the configs etc. Thanks for the pointer; perhaps we can sugar the configuration-less case to just bootstrap with prometheus client as parameter but that's minor...

Thanks for the pointer :)

Reply:
No problem. And definitely open to add sugar here 👍🏻